docs/en/stack/ml/anomaly-detection/ootb-ml-jobs-nginx.asciidoc

```diff
@@ -6,10 +6,89 @@
 ++++
 
 // tag::nginx-jobs[]
-These {anomaly-job} wizards appear in {kib} if you use {filebeat} to ship access 
-logs from your http://nginx.org/[Nginx] HTTP servers to {es} and store it using 
-fields and datatypes from the Elastic Common Schema (ECS). For more details, see
-the {dfeed} and job definitions in
+
+These {anomaly-job} wizards appear in {kib} if you use either {filebeat} or the
```

Comment:
You should switch the order of the filebeat and fleet integrations here, as you did for the Apache module.

Reply:
Done, thanks!